charts/kubermatic-operator/crd/k8c.io/operatingsystemmanager.k8c.io_operatingsystemconfig.yaml

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This file has been copied from
+# https://github.com/kubermatic/operating-system-manager/blob/master/deploy/crd/crd-operating-system-config.yaml
```

Comment:
This is a minor nitpick, but should this perhaps be a permalink / link to a tag? That would make it easier to check "upstream" if the CRD changed since the last time it was copied to kubermatic/kubermatic.

Reply:
The "check if upstream changed" would be "copy the new files, check git diff" IMHO. I'm not even sure there are stable, tagged versions of OSM CRDs we could use?

Reply:
Shouldn't the CRD version in the release tag corresponding to the OSM release we deploy be the "stable" version, since it's in sync with what the OSM version we deploy expects? The `master` CRD might already diverge from what the release needs. No big deal I think, just a thought I had.